stl/inc/type_traits

```diff
@@ -1658,12 +1658,11 @@ struct _Invoker_ret<_Unforced, false> { // selected for _Rx being _Unforced
     }
 };
 
-

```

Comment:
This change is also being made in #865.

Reply:
I'll revert this here, but I suspect 1663 is close enough that we'll get a merge conflict anyway.